src/org/parosproxy/paros/extension/option/SecurityProtocolsPanel.java

```diff
@@ -114,10 +114,12 @@ public void setSecurityProtocolsEnabled(String[] selectedProtocols) {
             checkBox.setSelected(false);
         }
 
-        for (String protocol : selectedProtocols) {
-            JCheckBox checkBox = checkBoxesSslTlsProtocols.get(protocol);
-            if (checkBox != null && checkBox.isEnabled()) {
-                checkBox.setSelected(true);
+        if (selectedProtocols != null) {
```

Comment:
Can it actually be null? Do we need a log message or something if it is? (Because there won't be any checkboxes?)

Reply:
I hit this when testing - I think its just when initializing in some cases, and it doesnt actually cause any problems.

Reply:
Ok